packages/xchain-bitcoin/src/utils.ts

```diff
@@ -147,7 +140,7 @@ export const scanUTXOs = async ({
   network,
   address,
   confirmedOnly = true, // default: scan only confirmed UTXOs
-}: ScanUTXOParam): Promise<UTXOs> => {
+}: ScanUTXOParam): Promise<UTXO[]> => {
```

Comment:
Because `export type UTXOs = UTXO[]` isn't very useful; it doesn't describe anything  It only saves a single character, it requires you to import two symbols instead of one, and there's no case where you could change that definition to anything else without breaking downstream code which relies on stuff like `.map()`, `.reduce()`, `Array.isArray()`, etc. It might have a reason to exist if it was `export type UTXOs = Iterable<UTXO>`, but that's not what it is.